koku/forecast/test/tests_forecast.py

```diff
@@ -428,6 +429,13 @@ def test_enumerate_dates(self):
                 out = instance._enumerate_dates(scenario["dates"])
                 self.assertEqual(out, scenario["expected"])
 
+    def test_summary_table(self):
+        """COST-908: Test that the expected summary table is used."""
+        mock_access = {"aws.organizational_unit": {"read": ["1234", "5678"]}}
+        params = self.mocked_query_params("?", AWSCostForecastView, access=mock_access)
+        instance = AWSForecast(params)
+        self.assertEqual(instance.cost_summary_table, AWSCostEntryLineItemDailySummary)
```

Comment:
@blentz In the case that we have RBAC restriction only on org unit, we should actually be getting the `reporting_aws_cost_summary_by_account` materialized view, so I think we actually want to update the views attribute in the AWS provider map.